.github/workflows/ci.yaml

```
@@ -0,0 +1,31 @@
+        shell: bash
+        run: find . -type f -name ".env.example" -exec sh -c 'cp "$1" "${1%.*}"' _ {} \;
+
+      
+      - name: Create dummy amplify_outputs.json for CI
+        run: echo '{}' > ./amplify_outputs.json
+
+      - name: Typecheck
+        run: pnpm typecheck
+
+      - name: Lint
+        run: pnpm lint
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)